docs/openebs-mixin-user-guide.md

```diff
@@ -0,0 +1,129 @@
+# Install openebs-mixin
+
+---
+### Different scenarios to use openebs-mixin
+
+1. Install openebs-mixin with openebs-monitoring written in jsonnet.
+2. Install openebs-mixin in existing prometheus stack.
+
+---  
+## Install openebs-mixin with openebs-monitoring written in jsonnet.
+
+### Prerequisite
+
+- OpenEBS installed
+
+#### Step by Step Process
+
+1. **Configuration**  
+   The available fields and their default values are present in [config.libsonnet](../jsonnet/config.libsonnet). To update the configuration of mixin, you need to override the configuration through config.libsonnet.
+	```
+   . . .
+	// import openebs-mixin and override configurations
+	openebsMixin: (import './openebs-mixin/mixin.libsonnet') {
+		_config+:: {
+			dashboards+: {
+				cStor: $._config.casType.cStor.dashboards,
+				jiva: $._config.casType.jiva.dashboards,
+				localPV: $._config.casType.lvmLocalPV.dashboards,
+				lvmLocalPV: $._config.casType.lvmLocalPV.dashboards,
+				deviceLocalPV: $._config.casType.lvmLocalPV.dashboards,
+			},
+			alertRules+: {
+				cStor: $._config.casType.cStor.alertRules,
+				jiva: $._config.casType.jiva.alertRules,
+				lvmLocalPV: $._config.casType.lvmLocalPV.alertRules,
+				deviceLocalPV: $._config.casType.lvmLocalPV.alertRules,
+			},
+		},
+	},
+  	. . .
+
+	```  
+
+2. **Install openebs-monitoring**  
```

Comment:
Does this step also install the mixin? Its a bit unclear as to what is meant by "install openebs-monitoring"